services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -3861,7 +3859,9 @@ protected String metricMessage(String userId,
                                  String operation,
                                  String status,
                                  Long duration,
-                                 String error) {
+                                 String error,
+                                 Double fileSize,
+                                 Integer uploadStatus) {
```

Comment:
as we have the string status, we do not need this last param. 
Please remove it